src/admin/content-block/components/wrappers/MediaGridWrapper/MediaGridWrapper.tsx

```diff
@@ -56,6 +74,73 @@ const MediaGridWrapper: FunctionComponent<MediaGridWrapperProps> = ({
 		[history]
 	);
 
+	const mapSearchData = useCallback(
+		(searchData: Avo.Search.Search): MediaListItem[] => {
+			if (searchData.results.length > 0) {
+				return searchData.results.map(searchItem => {
+					const category = toEnglishContentType(searchItem.administrative_type);
+					const isItem = ['audio', 'video'].includes(category);
+					const duration = formatDurationHoursMinutesSeconds(
+						get(searchItem, 'duration_seconds', 0)
+					);
+
+					return {
+						category,
+						metadata: [
+							{ icon: 'eye', label: String(searchItem.views_count) },
+							{ label: formatDate(searchItem.dcterms_issued) },
+						],
+						navigate: () =>
+							navigateToContentType(
+								{
+									type: isItem
+										? 'ITEM'
+										: (category.toUpperCase() as ContentPickerType),
+									value: searchItem.external_id,
+								},
+								history
+							),
+						title: searchItem.dc_title,
+						thumbnail: {
+							label: searchItem.administrative_type,
+							meta: isItem ? duration : '', // Amount of items in collection not present in search
+							src: searchItem.thumbnail_path || '',
+						},
+					};
+				});
+			}
+
+			return [];
+		},
+		[history]
+	);
+
+	useEffect(() => {
+		const fetchQueryAndMapData = async () => {
+			if (searchQuery && searchQuery.value) {
+				let valueObj;
+
+				// Wre have to wrap JSON.parse in this try..catch because if the value is cleared
+				// it returns an invalid JSON string which causes a crash
+				try {
+					valueObj = JSON.parse(searchQuery.value as string);
+				} catch (err) {
+					console.error(new CustomError('Failed to parse search query value', err));
```

Comment:
mss deze console.error enkel uitvoeren als searchQuery.value niet null/undefined is
en in dat geval ook een toast tonen aan de gebruiker, of zou die dan teveel getriggert worden bij iedere keyUp event?

Reply:
Deze wordt gewrapped in een if dus wordt idd enkel uitgevoerd als de `searchQuery.value` ingevuld is.
Toast wordt al getoond door de ContentPicker zelf.